include/itkVkComplexToComplex1DFFTImageFilter.h

```diff
@@ -98,8 +115,8 @@ class VkComplexToComplex1DFFTImageFilter : public ComplexToComplex1DFFTImageFilt
   PrintSelf(std::ostream & os, Indent indent) const override;
 
 private:
+  bool     m_UseVkGlobalConfiguration = true;
```

Comment:
We could match the following line's style by using
```cpp
  bool     m_UseVkGlobalConfiguration{ true };
```

... and similarly in the other places where this occurs.